### DISCUSSION TITLE
Change Nnn default mapping to open a file

### DISCUSSION THREAD

**Discussion by kyoz** [mcchrish/nnn.vim#149] [Q&A] [answered]
By default, we can use Enter to open a file and o with let us select an app to open that file.

I'v used some plugins before and get comfortable with:
- `o` or `l` to open a file
- `Shift - o` to open a file with...

I'v tried to config like this:

```
let g:nnn#action = {
      \ '<S-t>': 'tab split',
      \ '<S-v>': 'split',
      \ '<S-h>': 'vsplit'
      \ 'o'    : ':e' ,
      \ 'l'    : ':e' 
```

It some how work for `o` and `l` key to open a file,  but when creating a new file with `n` or inside terminal mode, press `o` or `l` with close them :'(

Anyone has solution for this.

**Comment by mcchrish** [comment, marked as answer]
That's currently a limitation.

Meanwhile, I think nnn itself does not have a feature to create custom mapping which could have been handy with your use-case.
